src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -16,23 +16,20 @@
 
 
 (react/defc DatasetsTable
-  {:set-filter-text
-   (fn [{:keys [refs]} new-filter-text]
-     (react/call :update-query-params (@refs "table") {:filter-text new-filter-text :current-page 1}))
-   :render
+  {:render
    (fn [{:keys [state this props]}]
      (let [attributes (:library-attributes props)
            search-result-columns (:search-result-columns props)
            extra-columns (subvec search-result-columns 4)]
        [table/Table
-        {:ref "table" :state-key "library-table" :v 1
+        {:ref "table" :state-key "library-table" :v 2
```

Comment:
can we add some constants for the name and version number similar to what we did below for library page?